plugins/filter_parser/filter_parser.c

```diff
@@ -340,6 +343,30 @@ static int cb_parser_filter(const void *data, size_t bytes,
                         return FLB_FILTER_NOTOUCH;
                     }
 
+                    out_buf = new_buf;
+                    out_size = new_size;
+                }
+                else {
+                    out_buf = strdup(parsed_buf);
+                    out_size = parsed_size;
+                }
+                if (ctx->hash_value_field!=NULL) {
```

Comment:
Could you put a space around the not equal symbol like this?

```
if (ctx->hash_value_field != NULL) {
```

Reply:
Sure, is there any autoformatter configured? I can use itbto avoid these